site/docs/src/json/entity-types/response.json

```diff
@@ -0,0 +1,31 @@
+{
+  "entityType": {
+    "data": {
+      "createdBy": "jared@fusionauth.io",
+    },
+    "id": "8174f72f-5ecd-4eae-8de8-7fef597b3473",
+    "insertInstant": 1595361142909,
+    "jwtConfiguration": {
+      "accessTokenKeyId": "a7516c7c-6234-4021-b0b4-8870c807aeb2",
+      "enabled": true,
+      "timeToLiveInSeconds": 3600
+    },
+    "lastUpdateInstant": 1595361143101,
+    "name": "Customer",
+    "permissions": [
+      {
+        "entityTypeId": "8174f72f-5ecd-4eae-8de8-7fef597b3473",
+        "id": "ce485a91-906f-4615-af75-81d37dc71e90",
+        "insertInstant": 1595361142909,
+        "isDefault": false,
+        "lastUpdateInstant": 1595361143101,
+        "name": "admin"
+      },
+      {
+        "id": "ce485a91-906f-4615-af75-81d37dc71e91",
+        "name": "user",
+        "isDefault": true
+      }
```

Comment:
`insertInstant` and `lastUpdateInstant` are required so I do not think we should omit them here.